src/presentation/http/router/note.test.ts

```diff
@@ -192,4 +199,58 @@ describe('Note API', () => {
 
     test.todo('API should not return internal id and "publicId".  It should return only "id" which is public id.');
   });
-});
\ No newline at end of file
+});
+
+describe('Access rights', () => {
```

Comment:
This describe section should be included at the `getNoteBuId` describe because it is related only to this method. Describe-suits could be nested several times, it's ok for better tests organizing